templates/hiera.yaml.epp

```diff
@@ -61,7 +61,15 @@ hierarchy:
 <% if $hr['options'] { -%>
     options:
     <%- $hr['options'].each |$key,$value| { -%>
+    <%- if $value =~ Array { -%>
```

Comment:
This block (through L72, I think) should have an extra indent.

Reply:
ok, I made an extra indent.